backend/core/schedule/test_team_shopping_list.py

```diff
@@ -12,11 +11,11 @@ def test_fetching_team_shopping_list(client, user, team, recipe):
     end = start + timedelta(days=1)
     params = {"start": start, "end": end}
 
-    url = reverse("team-shoppinglist-list", kwargs={"team_pk": team.pk})
+    url = f"/api/v1/t/{team.pk}/shoppinglist/"
```

Comment:
Is there a test for the `me` and "pk" versions?

Reply:
https://github.com/chdsbd/recipeyak/blob/c0a54f005e1c3a6551451fa21fc00a1349726ccf/backend/core/schedule/test_shopping_list.py#L21-L23